Mage.Tests/src/test/java/org/mage/test/cards/single/_40k/AnrakyrTheTravellerTest.java

```diff
@@ -0,0 +1,186 @@
+package org.mage.test.cards.single._40k;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+public class AnrakyrTheTravellerTest extends CardTestPlayerBase {
+
+    private static final String ANRAKYR = "Anrakyr the Traveller";
+
+    @Test
+    public void testCastArtifactCardForLife_ShouldCastForLife() {
+        // prepare
+        final String cryptek = "Cryptek";
```

Comment:
I'd recommend moving these to private static final like ANRAKYR, rather than declaring them in each test, if they're used in more than one test. Also, please include rules text for cards used as a comment